### PR TITLE
enahnce purge dups

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/purge_dups/hist_plot.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+
+def col_hist(stat_fn, delim):
+    hists = []
+    # we consider the coverage histogram start with 0
+    with open(stat_fn) as f:
+        for ln in f:
+            lnlist = ln.strip().split(delim)
+            hists.append(int(lnlist[1]))
+        f.close()
```
Comment (mvdbeek) [inline tools/purge_dups/hist_plot.py:21]:
```suggestion
```

File: tools/purge_dups/purge_dups.xml [hunk 2]
```diff
@@ -232,15 +286,17 @@
...
-            <filter>function_select['functions'] == 'pbcstat'</filter>
+
+        <data name="hist" format="png" from_work_dir="hist.png" label="${tool.name} on ${on_string}: histogram plot">
+            <filter>function_select['functions'] == 'pbcstat' or function_select['functions'] == 'ngscstat'</filter>
         </data>
+
         <!-- Calcuts -->
         <data name="calcuts_log" format="txt" from_work_dir="calcuts.log" label="${tool.name} on ${on_string}: calcuts log file">
-            <filter>function_select['functions'] == 'calcuts'</filter>
+            <filter>function_select['functions'] == 'pbcstat' or function_select['functions'] == 'ngscstat' or function_select['functions'] == 'calcuts'</filter>
```
Comment (mvdbeek) [inline tools/purge_dups/purge_dups.xml:296]:
```suggestion
            <filter>function_select['functions'] in ('pbcstat', 'ngscstat', 'calcuts')</filter>
```

File: tools/purge_dups/purge_dups.xml [hunk 3]
```diff
@@ -232,15 +286,17 @@
...
         </data>
         <data name="calcuts_tab" format="tabular" from_work_dir="cutoffs.tsv" label="${tool.name} on ${on_string}: calcuts cutoff file">
-            <filter>function_select['functions'] == 'calcuts'</filter>
+            <filter>function_select['functions'] == 'pbcstat' or function_select['functions'] == 'ngscstat' or function_select['functions'] == 'calcuts'</filter>
```
Comment (mvdbeek) [inline tools/purge_dups/purge_dups.xml:299]:
```suggestion
            <filter>function_select['functions'] in ('pbcstat', 'ngscstat', 'calcuts')</filter>
```

**Review by mvdbeek — APPROVED** [review]
I'd prefer a separate plotting tool, but looks like this had always been the intention ?
Otherwise looks good.

**Comment by bgruening** [comment]
Thanks for the review. The plot is too specific for the tool and is kind of a quality control output. I would prefer to leave it in and gather some feedback from the VGP people. Maybe long term we can make it more agnostic and turn this into a proper tool.

Thanks for the review @mvdbeek

**Pull request merged**